website/docs/r/is_virtual_endpoint_gateway.html.markdown

```diff
@@ -77,6 +81,7 @@ Review the argument references that you can specify for your resource.
   **NOTE**: `id` and `subnet` are mutually exclusive.
 
 - `resource_group` - (Optional, Forces new resource, String) The resource group ID.
+- `security_groups` - (Optional, list) The security groups to use for this endpoint gateway.
```

Comment:
Add in the description - If unspecified, the VPC's default security group is used.